browser/lib/util/http.ts

```diff
@@ -0,0 +1,176 @@
+import * as Utils from '../../../common/lib/util/utils';
+import Defaults from '../../../common/lib/util/defaults';
+import ErrorInfo from '../../../common/lib/types/errorinfo';
+import { ErrnoException, IHttp, RequestCallback, RequestParams } from '../../../common/types/http';
+import HttpMethods from '../../../common/constants/HttpMethods';
+
+// TODO: replace these with the real types once these classes are in TypeScript
```

Comment:
Can this comment be removed now?

Reply:
[ce064d0](https://github.com/ably/ably-js/pull/762/commits/ce064d0ddb1ca40d21d618c425ec9f9eb42de62b)